andela/trime
# Fix broken navigation

### What does this PR do?

Fixes removing change listeners from the ProjectPageContainer and
the TrackingWidgetContainer.

### How should this be manually tested?

Go to the project page and then try navigating back to the org page. The navigation should work.

### What are the relevant pivotal tracker stories?
[#121020425](https://www.pivotaltracker.com/story/show/121020425)
